components/reference-layout/server.js

```diff
@@ -16,9 +16,8 @@ export class ReferenceLayout extends ServerComponent {
    */
   render(context) {
     const { doc } = context;
-    const [description, ...sections] = doc.body.map((section) =>
-      ContentSection.render(context, section),
-    );
+    const [description, ...sections] =
```

Comment:
(Unrelated, but the names `description` and `sections` here confused me a bit. We could just call them `firstSection` and `otherSections`.)